domains/client/domain-operator/src/tests.rs

```diff
@@ -3413,7 +3413,6 @@ async fn existing_bundle_can_be_resubmitted_to_new_fork() {
 // TODO: this test is flaky and may hang forever in CI, able it after the root cause is
 // located and fixed.
 #[tokio::test(flavor = "multi_thread")]
-#[ignore]
```

Comment:
There are 2 more tests `test_unordered_cross_domains_message_should_work` and `test_cross_domains_message_should_work` were commented out in the code, plz also enable them (or remove them if they are outdated) as XDM is stable and soon release.